templates/admin/referent/admin_list.html.twig

```diff
@@ -0,0 +1,6 @@
+{% extends '@SonataAdmin/CRUD/base_list.html.twig' %}
+
+{% block pager_links %}
+    <div style="margin-top: 10px;">Faites dérouler ce dropdown pour exporter les équipes sélectionnées.</div>
```

Comment:
Why do you add css in style instead of css files ?

Reply:
Because this template overrides a Sonata template that does not use the application css files. To use the existing css files we should add a link to css file, the inline css is lighter. We use this practice for Sonata templates in the project.